components/page-header/page-header.js

```diff
@@ -2,56 +2,143 @@ import React from 'react';
+		}
+	}
+
+	animateLayer(scrolledHeight) {
+		// don't animate when component is hidden
+		if (this.isHidden) { return; }
+
+		// set opacity
+		const opacity =  1 - (scrolledHeight / this.range);
+
+		// set styles to animate
+		const styles = {
+			y: scrolledHeight * this.speed,
+			opacity
+		};
+
+		// animate styles with tweenlight
+		TweenLite.to(this.parallaxLayer, 0, styles, {ease: "Linear.easeNone" });
+	}
+
+	showVideo() {
+		this.setState({showVideo: true});
+	}
+
+	render() {
+		const props = this.props;
+		const imageParameters = { fit: 'max', fm: 'jpg', q: 90 }
+		const heroImageSmall = `${setImageParams(props.image, {...imageParameters, w: 768} )}`;
+		const heroImageMedium = `${setImageParams(props.image, {...imageParameters, w: 1170} )}`;
+		const heroImageLarge = `${setImageParams(props.image, {...imageParameters, w: 1244} )}`;
+		
+		const style ={__html:
+			`<style>
+				.page-header {
+					background-image: url(${heroImageSmall});
 				}
-			}
-			@media only screen and (min-width: 1170px) {
-				.page-header-inner {
-					background-image: url(${heroImageLarge});
+				@media only screen and (min-width: 768px) {
+					.page-header {
+						background-image: url(${heroImageMedium});
+					}
 				}
-			}
-		</style>`};
-
-
-	return (
-		<div className="page-header container">
-			{parallaxLayerBack}
-			<div className="page-header-overlay">
-				<div className="container-inner page-header-inner">
-					<h1 className="page-header-heading content">{title}</h1>
-
-					<button className="page-header-button content"
-							onClick={onClickScrollButton ? onClickScrollButton : null} >
-						<span className="page-header-button-text">{subtitle}</span>
-						<span className="icon">
-							<Icon icon="arrowDownCircle" />
-						</span>
-					</button>
-				</div>
-			</div>
-			{parallaxLayerFront}
+				@media only screen and (min-width: 1170px) { 
+					.page-header {
+						background-image: url(${heroImageLarge});
+					}
+				}
+			${props.video ?
+				`@media only screen and (min-width: 768px) {
+					.page-header {
+						background-image: none;
+					}
+				}` : '' }
+			}		
+			</style>`};
 
+		return (
+			<section
+				ref={node => this.element = node}
+				className={`page-header
+				${props.type === 'small' ? 'page-header-small' : ''} 
+				${this.state.showVideo ? 'show-video': ''}`}>
+				{ props.video &&
+					<video ref={node => this.video = node}
+					   	className="page-header-video"
+						playsInline autoPlay muted loop>
+						<source src={props.video} type="video/mp4" />
+					</video>
+				}
 
-			<div dangerouslySetInnerHTML={style} />
-		</div>
-	);
+				<div className="page-header-inner container">
+					<div ref={node => this.parallaxLayer = node}>
+						<h1 className="page-header-title ">{props.title}</h1>
+						<p className="page-header-subtitle">{props.subtitle}</p>
+						{ props.onClickScrollButton && 
+							<button className={`page-header-button 
+												${props.type === 'small' ? 
```

Comment:
I think its better add just one class on the section element and style on that. Instead of adding multiple classes in the component when its small. So to style this element: 
` .page-header-button { 
.page-header-small & {
// styling here
}}
`